package.json

```diff
@@ -28,10 +28,11 @@
     "async": "^2.6.0",
     "demolish": "^1.0.2",
     "diagnostics": "^1.0.1",
+    "fetch-retry": "^4.0.1",
+    "node-fetch": "^2.6.1",
     "optional-options": "^1.0.0",
     "out-of-band-cache": "^1.1.0",
     "querystringify": "^2.1.1",
-    "request": "^2.70.0",
     "retryme": "^1.0.0"
```

Comment:
we also have this retry library, should everything just use that or should we switch over everything to the `fetch-retry` library.

Reply:
I took a stab and swapping to use `retryme` for this, but it made the code a whole lot uglier. I like how `fetch-retry` just encapsulates the retries nicely. `retryme` is better for more complex operations, though, so I think it's fine to keep both.